mon-pix/app/routes/update-expired-password.js

```diff
@@ -8,7 +8,7 @@ export default class UpdateExpiredPasswordRoute extends Route.extend(Unauthentic
 
   model() {
     const users = this.store.peekAll('user');
-    const user = users.firstObject;
+    const user = users.find(({ id }) => id === null);
```

Comment:
**question**
En définitive, c'est quoi ce user ? C'est lequel, comment le nommer pourqu'on explicite l'intention ?

```suggestion
    const user = users.find(({ id }) => id === null);
```

Reply:
Laisser `user` ne me dérange pas, mais s'il faut être plus précis je propose 

```suggestion
    const userWithExpiredPassword = users.find(({ id }) => id === null);
```

Reply:
Done